ton_client/src/debot/action.rs

```diff
@@ -36,20 +36,30 @@ impl Default for AcType {
     fn default() -> Self { AcType::Empty }
 }
 
+/// Describes a debot action in a Debot Context.
 #[derive(Serialize, Deserialize, Clone, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct DAction {
+    /// a short action description. Should be used by Debot Browser as name of
+    /// menu item.
     #[serde(deserialize_with = "from_hex_to_utf8_str")]
     pub desc: String,
+    /// Depends on action type. Can be a debot function name or a print string 
+    /// (for Print Action).
     #[serde(deserialize_with = "from_hex_to_utf8_str")]
     pub name: String,
+    /// Action type. See structure AcType.
```

Comment:
Don't mention `AcType` type. It is linked automatically

Reply:
Ok